FinalEngine.Editor.Desktop/Views/Inspectors/EntityComponentView.xaml

```diff
@@ -0,0 +1,84 @@
+<UserControl x:Class="FinalEngine.Editor.Desktop.Views.Inspectors.EntityComponentView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:selectors="clr-namespace:FinalEngine.Editor.Desktop.Selectors.Data.Editing"
+             xmlns:dtv="clr-namespace:FinalEngine.Editor.Desktop.Views.Editing.DataTypes"
+             xmlns:dt="clr-namespace:FinalEngine.Editor.ViewModels.Editing.DataTypes;assembly=FinalEngine.Editor.ViewModels"
+             xmlns:vm="clr-namespace:FinalEngine.Editor.ViewModels.Inspectors;assembly=FinalEngine.Editor.ViewModels"
+             d:DataContext="{x:Type vm:EntityComponentViewModel}"
+             mc:Ignorable="d">
+    <Grid Margin="5">
+        <StackPanel>
+            <ToggleButton Content="{Binding Name}" Command="{Binding ToggleCommand}" />
+
+            <ListBox x:Name="ItemsListBox" ItemsSource="{Binding PropertyViewModels, UpdateSourceTrigger=PropertyChanged}"
+                     ScrollViewer.VerticalScrollBarVisibility="Auto"
```

Comment:
The scroll viewer is a little odd with the ListBox; the scroll bar will scroll to each individual component instead of being smooth. I should instead setup a scroll viewer for the tool view potentially.